package.json

```diff
@@ -61,8 +62,9 @@
     "@typescript-eslint/parser": "^3.10.1",
     "broccoli-asset-rev": "^3.0.0",
     "documentation": "^13.0.2",
+    "ember-auto-import": "^2.4.2",
     "ember-cli": "~3.21.2",
-    "ember-cli-addon-docs": "^0.8.0",
+    "ember-cli-addon-docs": "^4.2.2",
```

Comment:
I'm not sure we should be bumping addon-docs all the way up to this version. I'd prefer we do minimal changes in this release branch, and defer changes to the master branch.